### PR TITLE
Finished Badge and Button Components

### PR DESCRIPTION
**Pull request by Naman-Saxena1**

### CODE REVIEW

File: Components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,411 @@
...
+    flex-wrap: wrap;
+    padding: 3rem 1rem;
+}
+
+
+/* Primary Solid Buttons */
+.solid-primary-btn
+{
+    min-width: 120px;
+    min-height: 45px;
```
Comment (anasm11) [inline Components/Button/button.css]: Better to give lengths in rems. Rest all looks good.

File: Components/Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+.count-badge-5x
+{
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color:  var(--notification-or-error);
+    border-radius: 100%;
+    color: white;
```
Comment (bhtibrewal) [inline Components/Badge/badge.css]: - Is Border radius 100% needed (50% should suffice)? 
- avoid using pure white/black and writing color name

File: Components/Badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+    width: 1.5rem;
+    height: 1.5rem;
+    top: -20px;
+    right: -20px;
+    font-size: 15px;
```
Comment (bhtibrewal) [inline Components/Badge/badge.css]: avoid using px (line 49)

File: Components/Badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+    right: -12px;
+    font-size: 10px;
+}
+
+.status-badge-5x
+{
+    position: absolute;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```
Comment (bhtibrewal) [inline Components/Badge/badge.css]: redundant lines of code... you can refactor like this 
```
.satus-badge-5x, count-badge-2x, ...{
    position: absolute;
    display: flex;
    justify-content: center;
    align-items: center;
}
```

File: Components/Badge/badge.css [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+    background-color:  var(--onlinestatus-or-success);
+    border-radius: 100%;
+    color: white;
```
Comment (bhtibrewal) [inline Components/Badge/badge.css]: again avoid pure white/black and color name

File: Components/Badge/badge.css [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+}
+
+.status-online
+{
+    background-color:  var(--onlinestatus-or-success);
+}
+
+.status-offline
+{
+    background-color:  rgb(172, 172, 172);
```
Comment (bhtibrewal) [inline Components/Badge/badge.css]: replace this with the var name

File: Components/Button/button.css [hunk 7]
```diff
@@ -0,0 +1,411 @@
...
+    background: #ff5f53
+                radial-gradient(circle, transparent 1%, 
+                                    var(--notification-or-error) 1%) center/15000%;
+}
+
+.solid-error-btn:active 
+{
+    background-color: var(--notification-or-error);
+    background-size: 100%;
+    transition: background 0s;
```
Comment (bhtibrewal) [inline Components/Button/button.css:116]: use of this?

**Review by bhtibrewal — APPROVED** [review]
The UI design is good. I have written few reviews for the code, rest all looks good

File: components.css [hunk 8]
```diff
@@ -27,9 +27,9 @@
     --background-color        :#F3F3F3;
     --extra-orange-color      :#EF836D;
     --compilmentary-blue-color:#1AB3B3;
```
Comment (parvez-rashid) [inline components.css]: Typo - fix "compilmentary" to "complimentary"

**Review by Ankur9669 — CHANGES_REQUESTED** [review]
Buttons do not have cursor:pointer rest all looks fine.

File: Components/Button/button.css (hunk 1 above)
Comment (Naman-Saxena1) [reply Components/Button/button.css]: Done, Thanks for review.

File: Components/Badge/badge.css (hunk 2 above)
Comment (Naman-Saxena1) [reply Components/Badge/badge.css]: Done, Thanks for review.

File: Components/Badge/badge.css (hunk 4 above)
Comment (Naman-Saxena1) [reply Components/Badge/badge.css]: Done, Thanks for review. This advice was really helpful.

File: Components/Badge/badge.css (hunk 5 above)
Comment (Naman-Saxena1) [reply Components/Badge/badge.css]: Done, Thanks for review.

File: Components/Badge/badge.css (hunk 6 above)
Comment (Naman-Saxena1) [reply Components/Badge/badge.css]: Done, Thanks for review.

File: components.css (hunk 8 above)
Comment (Naman-Saxena1) [reply components.css]: Done, thanks for review.

**Comment by Naman-Saxena1** [comment]
> Buttons do not have cursor:pointer rest all looks fine.

Done, thanks for review.

File: Components/Button/button.css (hunk 7 above)
Comment (Naman-Saxena1) [reply Components/Button/button.css:116]: This one is needed for button click ripple animation, if I remove it then animation won't work and if I increase its time, animation will work way too late after button click, I wanted animation to start exactly on button click.

File: Components/Badge/badge.css (hunk 3 above)
Comment (Naman-Saxena1) [reply Components/Badge/badge.css]: Done, thanks for advice.

**Pull request merged**